app/bundles/EmailBundle/Views/Email/list.html.php

```diff
@@ -217,6 +250,9 @@
                         <?php echo $view['content']->getCustomContent('email.stats', $mauticTemplateVars); ?>
                         <?php echo $view['content']->getCustomContent('email.stats.below', $mauticTemplateVars); ?>
                     </td>
+                    <td class="visible-lg"><?php echo $item->getDateAdded() ? $view['date']->toFullConcat($item->getDateAdded()) : ''; ?></td>
+                    <td class="visible-lg"><?php echo $item->getDateModified() ? $view['date']->toFullConcat($item->getDateModified()) : ''; ?></td>
+                    <td class="visible-lg"><?php echo $item->getCreatedByUser(); ?></td>
```

Comment:
There is a chance for XSS. Let's not introduce new vulnerabilities.
```suggestion
                    <td class="visible-lg"><?php echo $view->escape($item->getCreatedByUser()); ?></td>
```

Reply:
I didn't notice it, thanks!